support/scripts/mkcpio

```diff
@@ -3,10 +3,37 @@ CPIOTOOL="$( which bsdcpio )"
 
 usage()
 {
-	echo "$0 [output.cpio] [CPIO root path]" 1>&2
+	echo "$0 [options] [output.cpio] [CPIO root path]" 1>&2
+	echo "-h	display this help screen" 1>&2
+	echo "-q	suppress non-critical output"
+	echo "-z	compress [output.cpio] using gzip to produce [output.cpio].gz" 1>&2
 	exit 1
 }
 
+# Process extra options
+FLAGS=""
+while getopts ":hqz" option; do
+	case $option in
+		h)
+			usage
+			;;
+		q)
+			FLAGS+="--quiet "
+			;;
+		z)
+			FLAGS+="-z "
+			;;
+		?)
+			usage
+			;;
+	esac
+done
+if [[ $FLAGS != *"--quiet"* ]]; then
```

Comment:
Maybe it is more efficient to work with an boolean like variable that you set within the case:
```bash
OPT_QUIET=1
# [...]

while getopts ":hqz" OPT; do
	case $OPT in
		# [...]
		q)
			OPT_QUIET=0
			;;
		# [...]
	esac
done

if [ $OPT_QUIET -eq 0 ]; then
	CPIOFLAGS+=("--quiet")
else
	CPIOFLAGS+=("-v")	
fi
```